src/conq/navigation_lib/openai_nav/openai_nav_client.py

```diff
@@ -91,3 +91,26 @@ def find_probable_location_for_object(self, object: str):
                 )
 
         return probabilities_out
+
+    @limits(calls=3, period=ONE_MINUTE)
+    def generate_label_for_image(self, prompt="Please describe the primary function of the following group of tools in a concise manner using no more than five words and avoid general terms like \"essential\" or \"gardening\".", image_url="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT354l_oPc87wCqcXwlLKCRPLfZUP-Cs3Wbzw&usqp=CAU"):
```

Comment:
I can't tell, does this `image_url` contain an API key or otherwise sensitive information?

Reply:
Ah yea it does look like that, but it is just an image link from google images for openai to default using